src/cdk/tsconfig-tests.json

```diff
@@ -15,6 +15,7 @@
   },
   "files": [
     "./testing/index.ts",
+    "./index.ts",
```

Comment:
Why has this been added?

Reply:
Otherwise the index.js is not generated when running the tests, leading to ‚cannot find module „@angular/cdk“‘

Reply:
Makes sense. For consistency it would be best to have it in the `include` below, but both ways are fine. Looks good.